bot/code_review_bot/tasks/coverage.py

```diff
@@ -86,6 +86,9 @@ class ZeroCoverageTask(AnalysisTask):
     route = "project.releng.services.project.production.code_coverage_bot.latest"
     artifacts = ["public/zero_coverage_report.json"]
 
+    # Do not send any warning when this task does not find any issues
+    is_allowed_empty = True
```

Comment:
Do we actually need this? The task here should always be green, so we shouldn't hit the condition anyway, no?

Reply:
We need this to keep a generic workflow: the zero code coverage task must be processed when it's green, but that's the only task that can be processed by the bot and not report any issue. Most other tasks are not processed as their state indicate that they did not find any issue

Reply:
Fixed in 6d7a6e5.